apps/minifront/src/components/swap/unclaimed-swaps.tsx

```diff
@@ -27,8 +27,8 @@ const _UnclaimedSwaps = ({ unclaimedSwaps }: { unclaimedSwaps: UnclaimedSwapsWit
   const { claimSwap, isInProgress } = useStore(unclaimedSwapsSelector);
 
   return (
-    <Card>
-      <GradientHeader>Unclaimed Swaps</GradientHeader>
+    <Card layout>
+      <GradientHeader layout>Unclaimed Swaps</GradientHeader>
```

Comment:
Non-blocking: can we get rid of this card entirely, and instead have unfinished swaps have a one-line layout and stack them on top of the auctions, so we never have to explain to users how this works?

Reply:
Will make an issue for this